src/components/GenericPagesComponents/HomePageTabImageNav/HomePageTabImageNav.tsx

```diff
@@ -3,16 +3,21 @@ import { OakFlex } from "@oaknational/oak-components";
 import Flex, { FlexProps } from "@/components/SharedComponents/Flex.deprecated";
 import HomePageTabImageButton from "@/components/GenericPagesComponents/HomepageTabImageButton";
 import { Hr } from "@/components/SharedComponents/Typography";
-import { HomePageTab } from "@/pages";
+
+export type HomePageTab = "teachers" | "curriculum" | "ai" | "pupils";
 
 const HomePageTabImageNav = ({
   current,
-  setCurrent,
   ...flexProps
 }: FlexProps & {
   current: HomePageTab | undefined;
-  setCurrent: (tab: HomePageTab) => void;
 }) => {
+  const setCurrent = (tab: HomePageTab) => {
+    if (tab === current) {
+      return;
+    }
+    window.location.href = `/${tab}`;
```

Comment:
can we change the buttons to links to the pages now?

Reply:
alas not without completely rewriting these janky buttons. Postponed to another ticket